internal/services/registry/storage_test.go

```diff
@@ -168,6 +181,9 @@ func (s *StorageTestSuite) Test_Mint_SyntheticDevice() {
 	).One(s.ctx, s.dbs.DBS().Reader)
 	s.NoError(err)
 
+	s.Equal(udi.UserDeviceID, udArgs.UserDeviceID)
```

Comment:
We could check all the relevant stuff, not just this field. I don't want to belabor this point, let's merge.